src/components/SubsidyRequestManagementTable/data/hooks.js

```diff
@@ -1,10 +1,125 @@
-import { useState, useEffect } from 'react';
+import { useCallback, useState, useEffect } from 'react';
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 import { logError } from '@edx/frontend-platform/logging';
+import debounce from 'lodash.debounce';
 
+import { transformRequestOverview, transformRequests } from './utils';
 import DiscoveryApiService from '../../../data/services/DiscoveryApiService';
 
-// eslint-disable-next-line import/prefer-default-export
+export const DEBOUNCE_TIME_MS = 200;
+export const PAGE_SIZE = 20;
+
+export const useSubsidyRequests = (
```

Comment:
Nice code reuse here 🏅 

Don't feel strongly about this but wonder if maybe we should pass some sort of enum to map to services? For example: 
```
const SUBSIDY_REQUESTS_TYPES = {
    'CODES': {
        overview: /* service reference here
        requests: /* service reference here
    },
    'LICENSES': {
        overview: /* service reference here
        requests: /* service reference here
    },
}
```
That way we only pass the enums `LICENSES` or `CODES` to `useSubsidyRequests` ensuring we are calling the right combination of services.